graphql/resolvers.go

```diff
@@ -3812,8 +3821,9 @@ func (r *versionResolver) BuildVariants(ctx context.Context, v *restModel.APIVer
 			{Key: task.DisplayNameKey, Order: 1},
 		}
 		opts := data.TaskFilterOptions{
-			Sorts:            defaultSort,
-			IncludeBaseTasks: true,
+			Sorts:                          defaultSort,
+			IncludeBaseTasks:               true,
+			IncludeBuildVariantDisplayName: false, // we don't need to include buildVariantDisplayName here because this is only used to determine if a task has been activated
```

Comment:
This is more of an observation rather than something to fix, but I kind of wonder why we don't use the function `setVersionActivationStatus` from `graphql/util.go` here. If you put them side by side, they're almost identical 

<img width="1696" alt="Screen Shot 2022-03-01 at 3 12 16 PM" src="https://user-images.githubusercontent.com/47064971/156241614-985526c0-5165-4362-b8d4-ed247db8c162.png">

Reply:
@malikchaya2  and I observed this as well. tbh they should live in the same spot and it was an oversight.  I opened this ticket earlier which should fix this issue. https://jira.mongodb.org/browse/EVG-16376

Reply:
sounds good to me 😇